src/backend/base/langflow/main.py

```
@@ -208,6 +208,9 @@
     from langflow.utils.version import get_version_info
 
     __version__ = get_version_info()["version"]
+    settings = get_settings_service().settings
+    root_path = settings.root_path
```

Ruff (0.11.9)

Local variable `root_path` is assigned to but never used

Remove assignment to unused variable `root_path`

(F841)